clang/lib/AST/Interp/ByteCodeExprGen.cpp

```diff
@@ -253,6 +253,29 @@ bool ByteCodeExprGen<Emitter>::VisitBinaryOperator(const BinaryOperator *BO) {
     return this->delegate(RHS);
   }
 
+  // Special case for C++'s three-way/spaceship operator <=>, which
+  // returns a std::strong_ordering (which is class, so doesn't have a
```

Comment:
So this isn't really true, see the rules here: https://en.cppreference.com/w/cpp/language/default_comparisons

It can end up returning a `std::weak_ordering` or `std::partial_ordering` as well.  The rest here is true, but I wanted to make sure you know, since these have slightly different semantics that I don't see handled here?

Reply:
I already put that up, but I see lower you refer to all 3, so this is just a request to modify that comment as necessary.

Reply:
Right, I wrote that comment for the first test case I had. Updated it, thanks.